### PR TITLE
HAL-1595: Configure support for TLS 1.3

### PR DESCRIPTION
**Pull request by hpehl**
Analysis document for https://issues.redhat.com/browse/HAL-1595

### CODE REVIEW

File: console/HAL-1595_Configure_support_for_TLS_13.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+:email:             hpehl@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.redhat.com/browse
+
+== Overview
+
+Add supprt for attribute `cipher-suite-names` of resource `/subsystem=elytron/server-ssl-context=*`.
```
Comment (fjuma) [inline console/HAL-1595_Configure_support_for_TLS_13.adoc:12]: @hpehl Support for the `cipher-suite-names` attribute should also be added to the `/subsystem=elytron/client-ssl-context=*` resource.

Comment (hpehl) [reply console/HAL-1595_Configure_support_for_TLS_13.adoc]: Thanks, @fjuma, for the hint! I've updated the analysis document.

Comment (hpehl) [reply console/HAL-1595_Configure_support_for_TLS_13.adoc]: Yeah, of course. Will do that just now.

File: console/HAL-1595_Configure_support_for_TLS_13.adoc [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+:email:             hpehl@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.redhat.com/browse
+
+== Overview
+
+Add supprt for attribute `cipher-suite-names` for resources `/subsystem=elytron/client-ssl-context=*` and `/subsystem=elytron/server-ssl-context=*`.
```
Comment (spriadka) [inline console/HAL-1595_Configure_support_for_TLS_13.adoc:12]: type-o

**Pull request merged**